File: java/source-eclipse/ListFolders/src/listfolders/includes/tree/TreeNode.java
package listfolders.includes.tree;

/*
 * Parent for DirNode and FileNode
 */
public class TreeNode {
  public String text;
  public String icon;
  
  public TreeNode(String text){
    this.text=text;
    this.icon = "./lib/images/directory.png";
  }
  
  public String toString(){
    return text;
  }
  
}

File: c#/source/List Folders/List Folders/Includes/Tree/TreeNode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ListFolders.Includes.Tree {
  public class TreeNode {

    public string text;
    public string icon;

    public TreeNode(string text) {
      this.text = text;
      this.icon = "./lib/images/directory.png";                     // default folder icon
    }

    public string toString() {
      return text;
    }

  }
}
